pkg/tiflowapi/tiflowapi.go

```diff
@@ -17,26 +17,41 @@ const (
 // MasterClient provides master server's api
 type MasterClient interface {
 	// GetMasters returns all master members from cluster
-	GetMasters() ([]*MastersInfo, error)
-	GetExecutors() ([]*WorkersInfo, error)
+	GetMasters() (MastersInfo, error)
+	GetExecutors() (ExecutorsInfo, error)
 	GetLeader() (LeaderInfo, error)
 	EvictLeader() error
 	DeleteMaster(name string) error
 	DeleteExecutor(name string) error
 }
 
-const leaderPrefix = "api/v1/leader"
+const (
+	leaderPrefix        = "api/v1/leader"
+	leaderResignPrefix  = "api/v1/leader/resign"
+	listMastersPrefix   = "api/v1/masters"
+	listExecutorsPrefix = "api/v1/executors"
+)
+
+type Master struct {
```

Comment:
i thinks those struc is json respne from tifllow engine? can we just import those struct?

Reply:
yes, but currently not doing that.